tsconfig.json

```diff
@@ -1,24 +1,109 @@
 {
   "compilerOptions": {
-    "lib": [
-      "es6"
-    ],
-    "target": "es6",
-    "moduleResolution": "node",
-    "module": "CommonJS",
-    "outDir": "./out",
-    "rootDir": "./src",
-    "sourceMap": true,
-    "esModuleInterop": true,
-  },
-  "include": [
-    "src/**/*"
-  ],
-  "exclude": [
-    "node_modules"
-  ],
-  "ts-node": {
-    "esm": true,
-    "experimentalSpecifierResolution": "node"
+    /* Visit https://aka.ms/tsconfig to read more about this file */
```

Comment:
please remove all commented lines from tsconfig

Reply:
fixed in [b5e081e]